r/vignettes/install.Rmd

```diff
@@ -157,11 +329,27 @@ something other than the RSPM binary mirror URLs).
 * Download the dependency files (`cpp/thirdparty/download_dependencies.sh` may be helpful)
 * Copy the directory of dependencies to the offline computer
 * Create the environment variable `ARROW_THIRDPARTY_DEPENDENCY_DIR` on the offline computer, pointing to the copied directory.
-* Install the `arrow` package as usual.
+* Install the arrow package as usual.
+
+# Installing libarrow dependencies
+
+When you install libarrow, its dependencies will be automatically downloaded.  
+The environment variable `ARROW_DEPENDENCY_SOURCE` controls whether the libarrow
+installation also downloads or installs all dependencies (when set to `BUNDLED`), 
+uses only system-installed dependencies (when set to `SYSTEM`) or checks 
+system-installed dependencies first and only installs dependencies which aren't 
+already present (when set to `AUTO`).
+
+These dependencies vary by platform; however, if you wish to install these 
+yourself prior to libarrow installation, we recommend that you take a look at 
+the [docker file for whichever of our CI builds](https://github.com/apache/arrow/tree/master/ci/docker)
+(the ones ending in "cpp" are for building Arrow's C++ libaries aka libarrow) 
+corresponds most closely to your setup.  This will contain the most up-to-date 
+information about dependencies and minimum versions.
```

Comment:
This section added in response to https://github.com/apache/arrow/pull/11881#pullrequestreview-825160605